ci-operator/step-registry/cucushift/installer/rehearse/aws/cases/multi-cidr/provision/cucushift-installer-rehearse-aws-cases-multi-cidr-provision-commands.sh

```diff
@@ -0,0 +1,916 @@
+      DestinationCidrBlock: 0.0.0.0/0
+      NatGatewayId:
+        Ref: NAT3
+  S3Endpoint:
+    Type: AWS::EC2::VPCEndpoint
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+        - Effect: Allow
+          Principal: '*'
+          Action:
+          - '*'
+          Resource:
+          - '*'
+      RouteTableIds:
+      - !Ref PublicRouteTable
+      - !Ref PrivateRouteTable
+      - !If [DoAz2, !Ref PrivateRouteTable2, !Ref "AWS::NoValue"]
+      - !If [DoAz3, !Ref PrivateRouteTable3, !Ref "AWS::NoValue"]
+      ServiceName: !Join
+      - ''
+      - - com.amazonaws.
+        - !Ref 'AWS::Region'
+        - .s3
+      VpcId: !Ref VPC
+  DhcpOptions: 
+    Type: AWS::EC2::DHCPOptions
+    Condition: DoDhcp
+    Properties: 
+        DomainName: example.com
+        DomainNameServers: 
+          - AmazonProvidedDNS
+  VPCDHCPOptionsAssociation:
+    Type: AWS::EC2::VPCDHCPOptionsAssociation
+    Condition: DoDhcp
+    Properties:
+      VpcId: !Ref VPC
+      DhcpOptionsId: !Ref DhcpOptions
+
+Outputs:
+  VpcId:
+    Description: ID of the new VPC.
+    Value: !Ref VPC
+  PublicSubnetIds:
+    Description: Subnet IDs of the public subnets.
+    Value:
+      !Join [
+        ",",
+        [!Ref PublicSubnet, !If [DoAz2, !Ref PublicSubnet2, !Ref "AWS::NoValue"], !If [DoAz3, !Ref PublicSubnet3, !Ref "AWS::NoValue"]]
+      ]
+  PrivateSubnetIds:
+    Description: Subnet IDs of the private subnets.
+    Value:
+      !Join [
+        ",",
+        [!Ref PrivateSubnet, !If [DoAz2, !Ref PrivateSubnet2, !Ref "AWS::NoValue"], !If [DoAz3, !Ref PrivateSubnet3, !Ref "AWS::NoValue"]]
+      ]
+  CIDRs:
+    Description: CIDRs.
+    Value:
+      !Join [
+        ",",
+        [!Ref VpcCidr, !If [DoAz2, !Ref VpcCidr2, !Ref "AWS::NoValue"], !If [DoAz3, !Ref VpcCidr3, !Ref "AWS::NoValue"]]
+      ]
+  SubnetsIdsForCidr:
+    Description: Subnet IDs for Cidr1, the first is private subnet, the second is public subnet
+    Value:
+      !Join [
+        ",",
+        [!Ref PrivateSubnet, !Ref PublicSubnet]
+      ]
+  SubnetsIdsForCidr2:
+    Description: Subnet IDs for Cidr2, the first is private subnet, the second is public subnet
+    Value:
+      !Join [
+        ",",
+        [!If [DoAz2, !Ref PrivateSubnet2, !Ref "AWS::NoValue"], !If [DoAz2, !Ref PublicSubnet2, !Ref "AWS::NoValue"]]
+      ]
+  SubnetsIdsForCidr3:
+    Description: Subnet IDs for Cidr3, the first is private subnet, the second is public subnet
+    Value:
+      !Join [
+        ",",
+        [!If [DoAz3, !Ref PrivateSubnet3, !Ref "AWS::NoValue"], !If [DoAz3, !Ref PublicSubnet3, !Ref "AWS::NoValue"]]
+      ]
+EOF
+
+CLUSTER_PREFIX="${NAMESPACE}-${UNIQUE_HASH}"
+STACK_NAME="${CLUSTER_PREFIX}-vpc"
+echo "${STACK_NAME}" > "${SHARED_DIR}/vpc_stack_name"
+
+EXPIRATION_DATE=$(date -d '6 hours' --iso=minutes --utc)
+TAGS="Key=expirationDate,Value=${EXPIRATION_DATE}"
+aws --region "${REGION}" cloudformation create-stack \
```

Comment:
Suggest to set some var, and assign `10.134.0.0/16`  and `10.190.0.0/16` to them, then pass them all related places as parameter, so that we can easily figure out how the cidr setting is passed down and tested.

Reply:
Done